### ISSUE TITLE
Create Login / Register form error colors and highlighting

### ISSUE THREAD

**Issue by Jieiku** [secretkeysio/jelly-actix-web-starter#31]
Currently when you leave a field blank in jelly you see this error:

[Value cannot be blank.] 

and it is the regular color instead of being highlighted red along with highlighting the field red, here is how amazon and most sites do it:

![2022-01-08_05-51](https://user-images.githubusercontent.com/106644/148646825-5eeaf996-b3af-4c41-8ef9-03edd7bf19a9.png)

This is what I am working on next.

**Comment by Jieiku** [comment]
still getting used to Tera templates.... this works but it seems like there should be way to do this with less lines of code....

**templates/accounts/register.html**:

```html
        {% if form.name.errors | length > 0 %}
            <label for="name">Your Name:</label>
            <input class=".error" type="text" name="name" value="{{ form.name.value }}">
        {% else %}
            <label for="name">Your Name:</label>
            <input type="text" name="name" value="{{ form.name.value }}">
        {% endif %}
        {% if form.name.errors | length > 0 %}
            {{ form.name.errors }}
        {% endif %}
```

**Comment by Jieiku** [comment]
mmm I also noticed that when you goto register. its does not seem to check all fields for error, it find the first field with an error and return the error. I need to see if I can change this so that ALL fields with an error are reported at the same time.

going to try and figure this out now.

**Comment by Jieiku** [comment]
There are these two files:
src/accounts/views/register.rs
templates/accounts/register.html

I see that the error reporting for the 3 registration fields are handled in separate files

name here:
jelly/src/forms/text.rs

email here:
jelly/src/forms/email.rs

password here:
jelly/src/forms/password.rs

I am assuming the errors are only returned on a first match basis because not all fields are checked simultaneously. Maybe the reason for this is theparate form handlers for text, email, password.

**Comment by ryanmcgrath** [comment]
Nah, this is easier than you think - just make it so this doesn't terminate early and validates all fields:

https://github.com/secretkeysio/jelly-actix-web-starter/blob/4a9120292b05380b79490f096f8cb7031ffad9dc/src/accounts/forms.rs#L31

**Comment by Jieiku** [comment]
at first I tried this:

```rs
impl Validation for NewAccountForm {
    fn is_valid(&mut self) -> bool {
        let name_isvalid:bool = self.name.is_valid();
        let email_isvalid:bool = self.email.is_valid();
        let password_isvlaid:bool = self.password.validate_with(&[&self.name, &self.email]);
        //self.name.is_valid() && self.email.is_valid() && self.password.validate_with(&[&self.name, &self.email])
        if name_isvalid == false || email_isvalid == false || password_isvlaid == false {
            return false;
        } else {
            return true;
        }
    }
}
```

but then lets say you fill in the username and password but not the name, then you get this:

![2022-01-13_17-32](https://user-images.githubusercontent.com/106644/149435927-cd1dfbcf-2e88-48b1-a29a-28cdcadc1e4c.png)

**Comment by Jieiku** [comment]
Thanks I see what I did wrong here.
